packages/node_modules/@webex/react-component-spaces-list/src/index.js

```diff
@@ -90,8 +92,9 @@ export default function SpacesList({
             className="ciscospark-spaces-list"
             height={height}
             rowCount={rowCount}
-            rowHeight={70}
+            rowHeight={52}
             rowRenderer={rowRenderer}
+            style={{backgroundColor: 'black'}}
```

Comment:
Can we use our imported styles or does this have to be an inline style?

Reply:
yeah i meant to switch it to an imported style, will do that